go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	github.com/CosmWasm/wasmd v0.31.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.2
-	github.com/cosmos/cosmos-sdk v0.47.2
+	github.com/cosmos/cosmos-sdk v0.47.3
```

Comment:
Is this sdk version bump intentional to have in this PR?

Reply:
Yeah this is automatic, and we should commit it since we replace at the end. Otherwise everyone's PR will keep showing this